inst/app/modules/plotly_testvisual.R

```diff
@@ -0,0 +1,7 @@
+output$plotly_testvisual = renderPlotly(
+  {
+    hk_accidents %>%
+      ggplot(aes(x = Type_of_Collision)) +
+      geom_histogram(stat = "count")
+  }
+)
```

Comment:
This is nice, Plotly has moved on since the last time I looked at it!

Reply:
Gosh, did you mean Plotly won't be updated anymore? What substitute package do you use?

Reply:
Oh no, I mean plotly has gotten better since I last used it in R (at least a year ago). 😅